### PR TITLE
Fix transfer amount setup

### PR DESCRIPTION
**Pull request by ERussel**
- setup initial value when provided during transfer initialisation

### CODE REVIEW

File: CommonWallet/Classes/Modules/Amount/AmountPresenter.swift [hunk 1]
```diff
@@ -93,7 +93,13 @@ final class AmountPresenter {
                                                           symbol: selectedAsset.symbol)
         assetSelectionViewModel.canSelect = account.assets.count > 1
 
-        amountInputViewModel = transferViewModelFactory.createAmountViewModel()
+        var optionalAmount: Decimal?
```
Comment (unklayman) [inline CommonWallet/Classes/Modules/Amount/AmountPresenter.swift]: Why "optional" in the name? It is obvious by the type

**Review by unklayman — APPROVED** [review]

File: CommonWallet/Classes/Modules/Amount/AmountPresenter.swift (hunk 1 above)
Comment (ERussel) [reply CommonWallet/Classes/Modules/Amount/AmountPresenter.swift]: Just a habit: at some point you might want to write something like if let amount = optionalAmount {}. But it is not the case here. I will fix it.

Comment (unklayman) [reply CommonWallet/Classes/Modules/Amount/AmountPresenter.swift]: `if let amount = amount {` works perfectly fine in such a case

Comment (ERussel) [reply CommonWallet/Classes/Modules/Amount/AmountPresenter.swift]: Sure. It works. I will commit a fix.

**Pull request merged**